tools/monitoring/clearalerts/main.go

```diff
@@ -54,6 +54,7 @@ func main() {
 		log.Fatalf("Failed to list the alerts: %v", err)
 	}
 
+	nDel := 0
```

Comment:
I dont think we need this var. We have the log statements for every pattern being deleted

Reply:
I can add a log statement `Finished clearing alerts in the database` at the end of the function instead of printing out the number of jobs deleted. This is mainly for printing something in the log so we know the job has completed successfully when there's no error pattern to delete.

Reply:
we can keep this then.